### PR TITLE
Add ch.gnugen.Moody

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

File: ch.gnugen.Moody.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "moody",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--talk-name=org.freedesktop.secrets"
+    ],
+    "build-options" : {
```
Comment (bilelmoussaoui) [inline ch.gnugen.Moody.json]: These build options are already set by default now.

File: ch.gnugen.Moody.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lexborisov/myhtml"
+                }
+            ],
+            "build-options" : {
+                "cflags" : "-O2 -g",
+                "cxxflags" : "-O2 -g",
```
Comment (bilelmoussaoui) [inline ch.gnugen.Moody.json]: Same

File: ch.gnugen.Moody.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        {
+            "name" : "moodle",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "config-opts" : [
+                "-Dprofiling=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
```
Comment (bilelmoussaoui) [inline ch.gnugen.Moody.json]: Nitpick: indentation is wrong here

**Comment by bilelmoussaoui** [comment]
bot, build ch.gnugen.Moody

File: ch.gnugen.Moody.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                    "url" : "https://github.com/lexborisov/myhtml"
+                }
+            ],
+            "build-options" : {
+                "cflags" : "-O2 -g",
+                "cxxflags" : "-O2 -g",
+                "env" : {
+                    "prefix" : "/app/"
```
Comment (barthalion) [inline ch.gnugen.Moody.json]: It's the default as well.

File: ch.gnugen.Moody.json [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                }
+            ]
+        },
+        {
+            "name" : "myhtml",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "make library",
+                "make install",
+                "cp *.pc /app/lib/pkgconfig/"
```
Comment (barthalion) [inline ch.gnugen.Moody.json]: You want to clean it up with `cleanup` array afterwards.

**Comment by barthalion** [comment]
Have you contacted upstream maintainer whether they're interested in co-maintaining their app on Flathub?

**Pull request closed without merging**